### PR TITLE
Sils/aroundoffixes

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

File: tests/js/ESLintBearTest.py [hunk 1]
```diff
@@ -37,3 +39,6 @@
 ESLintBearWithoutConfig = verify_local_bear(ESLintBear,
                                             valid_files=(test_good, test_bad),
                                             invalid_files=())
+
+ESLintBearSyntaxError = verify_local_bear(ESLintBear, valid_files=(),
```
Comment (AbdealiJK) [inline tests/js/ESLintBearTest.py]: Put this on top ? As this is a ESLintBearWithoutConfig test ?

File: package.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-    "complexity-report": "~2.0.0-alpha",
-    "csslint": "~0.10.0",
-    "dockerfile_lint": "~0.0.9",
-    "eslint": "~2.3.0",
-    "jshint": "~2.9.1",
-    "postcss-cli": "~2.5.1",
-    "remark": "~3.2.3",
-    "tslint": "~3.5.0"
+    "alex": "~2",
+    "autoprefixer": "~6",
```
Comment (AbdealiJK) [inline package.json]: Erm, I don';t think this is that you want.

Im not even sure what this means, but i'd guess that this means:
`~6` => `6` or `x` where `x` > 6

You want to do `~6.3`, which should mean `6.3` or `6.x` where `x` > 3

**Comment by AbdealiJK** [comment]
ack bf24498

File: package.json (hunk 2 above)
Comment (sils1297) [inline package.json]: ok removing this commit then

Comment (sils1297) [inline package.json]: although did you see my comment on the issue? I think ti's what we want

**Comment by sils1297** [comment]
reack bf24498

**Comment by sils1297** [comment]
TODO, add https://github.com/coala-analyzer/coala/issues/2125 to commit message

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Use >= for development versions so that source builds always work
-coala~=0.6.0
+coala>=0.7.0.dev20160508150317
```
Comment (Adrianzatreanu) [inline requirements.txt]: is this not failing anymore?

**Comment by Adrianzatreanu** [comment]
ack 8d50066

File: README.rst [hunk 4]
```diff
@@ -89,6 +89,11 @@ in this case.
 Usage
 -----
 
+::
+
+    echo "print('Hi!') " >> hw.py
```
Comment (Adrianzatreanu) [inline README.rst]: ``print('Hello World!')`` would be more accurate? :p

**Comment by Adrianzatreanu** [comment]
ack 75234f8de

**Comment by SanketDG** [comment]
I restarted the build

Old: https://circleci.com/gh/coala-analyzer/coala-bears/1220

File: README.rst (hunk 4 above)
Comment (sils1297) [inline README.rst]: wanted to keep it short

**Comment by Adrianzatreanu** [comment]
ack b1a4ae5c

**Comment by sils1297** [comment]
reack d33e6bd a5db561 9230d77

File: README.rst [hunk 5]
```diff
@@ -121,7 +121,9 @@ If you want to contribute to coala-bears, please take a look at the `Getting
 Involved Information
 <http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__.
 
-We appreciate any help!
+We appreciate any help! Join us on
+`gitter <https://gitter.im/coala-analyzer/coala/>`, IRC (#coala at freenode) or
+Telegram (https://telegram.me/joinchat/AuL-lwKZ8JLFZiI6SbtQVw).
```
Comment (SanketDG) [inline README.rst]: could you put these on separate bullets. Looks cleaner.
Also why the `<>` around gitter?

**Comment by SanketDG** [comment]
ack bfb2077

File: README.rst (hunk 5 above)
Comment (sils1297) [inline README.rst]: ah. oh. the gitter one is an rst link, telegram is not. Changing that.

**Comment by SanketDG** [comment]
ack a9e0068

**Comment by sils1297** [comment]
c9307fe README: Add bear listing examples
4d8a862 package.json: Allow minor version increases
ebeace9 README: Add minimal hello world example
2c1e6b4 README: Add pip debian information
1e0b92e ESLintBear: Don't show (None) if no rule specified
6e2e172 CPDBear: Set minimum tokens to 20
are ready

**Comment by SanketDG** [comment]
ack 56f981f

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**